android/src/main/java/com/mythichelm/localnotifications/factories/NotificationFactory.java

```diff
@@ -0,0 +1,73 @@
+package com.mythichelm.localnotifications.factories;
+
+import android.annotation.SuppressLint;
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Icon;
+import android.provider.Settings;
+
+import com.mythichelm.localnotifications.entities.NotificationAction;
+import com.mythichelm.localnotifications.entities.NotificationSettings;
+
+public class NotificationFactory implements INotifcationFactory {
+
+    @Override
+    public Notification createNotification(NotificationSettings settings, Context context) {
+        Icon applicationIcon = getApplicationIcon(context);
+
+        @SuppressLint("WrongConstant") Notification.Builder builder = new Notification.Builder(context)
+                .setContentTitle(settings.Title)
+                .setContentText(settings.Body)
+                .setSmallIcon(applicationIcon)
+                .setOngoing(settings.IsOngoing)
+                .setContentIntent(settings.ContentIntent)
+                .setPriority(settings.Priority)
+                .setSound(Settings.System.DEFAULT_NOTIFICATION_URI);
```

Comment:
I think something like this should be configurable. If it is too much work to support setting a custom sound from Dart, I think at least we should pass in a boolean like "notificationMakesSound" from Dart to enable/disable sound.

Reply:
I commented this line. We can add this back in with proper support later